api/controllers/EAppEligibilityQuestionsController.js

```diff
@@ -0,0 +1,120 @@
+const EAppEligibilityQuestionsController = {
+    renderQuestionOne(req, res) {
+        return EAppEligibilityQuestionsController._pageRendering(
+            req,
+            res,
+            'eApostilles/eligibilityQuestionOne.ejs'
+        );
+    },
+
+    renderQuestionTwo(req, res) {
+        return EAppEligibilityQuestionsController._pageRendering(
+            req,
+            res,
+            'eApostilles/eligibilityQuestionTwo.ejs'
+        );
+    },
+
+    renderQuestionThree(req, res) {
+        return EAppEligibilityQuestionsController._pageRendering(
+            req,
+            res,
+            'eApostilles/eligibilityQuestionThree.ejs'
+        );
+    },
```

Comment:
These routes look essentially the same. There are a few things most/all web frameworks will do to make this simpler/DRYer. (Suggest you read the comments on config/routes.js first since this will make more sense) Dynamic parameters will be useful here. 
https://sailsjs.com/documentation/concepts/routes/custom-routes#?wildcards-and-dynamic-parameters

using this (combined with comments in routes.js), I might have a route `/eligibility/:question`. So, when a user navigates to `service.gov.uk/eligibility/is-e-Apostille-accepted-in-destination`, I can do `req.param('question')`, and it will give me `is-e-Apostille-accepted-in-destination`. I can then combine with a condition object

```suggestion
    eligibilityQuestion(req, res) {
        const pages = {
          // "question-1" : "relevant-template.ejs" 
        }
        const question = req.param('question')
        return EAppEligibilityQuestionsController._pageRendering(
             req,
             res,
             pages[question]
         );
    },
```

it may not even be necessary to have this separate _pageRendering method if you're able to get the 3 methods down to one

Reply:
This is a really good point. I've never thought about passing dynamic values through the url for this but it does make a lot of sense. Hopefully shouldn't be too difficult to implement :)